modules/boss/src/main/ts/ephox/boss/mutant/Properties.ts

```diff
@@ -27,18 +27,22 @@ const document = (_item: Gene): undefined => {
 };
 
 const isText = (item: Gene): boolean => {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-enum-comparison
```

Comment:
I guess we could fix this as well later with a const record or something instead.

Reply:
Yeah, I think we need to evaluate our use of enums as they have gotchas that might not be worth the convenience of using them

Reply:
I have created https://ephocks.atlassian.net/browse/TINY-11765 as general investigation of our use of enums